server/athenian/api/controllers/settings_controller.py

```diff
@@ -493,35 +498,43 @@ async def set_logical_repository(request: AthenianWebRequest, body: dict) -> web
                             RepositorySet.updated_at: datetime.now(timezone.utc),
                             RepositorySet.items: items,
                         }))
-    del body["account"]
-    return model_response(WebLogicalRepository.from_dict(body))
+    return response
 
 
-async def _clean_logical_deployments(repo: str,
-                                     account: int,
-                                     pdb: DatabaseLike) -> None:
-    affected_deployments = await pdb.fetch_all(union(*(
-        select([distinct(model.deployment_name)]).where(and_(
-            model.acc_id == account,
-            model.repository_full_name == repo,
-        ))
-        for model in (GitHubCommitDeployment, GitHubPullRequestDeployment, GitHubReleaseDeployment)
+async def _find_matching_logical_repository(
+    logical_repo: LogicalRepository,
```

Comment:
`logical_repo` duplicates `logical_repo_request` in almost everything, not good. How about consuming `parent_name` and `ReleaseSettings` instead?

Reply:
done